tests/functional/pages/thank-you.page.js

```diff
@@ -5,6 +5,10 @@ class ThankYou {
     return url.indexOf('thank-you') > -1
   }
 
+  getTitle() {
```

Comment:
This may get confused with getting the contents of the HTML `<title>` element, maybe `getMainHeading` would be better?